course/templates/course/course_base.html

```diff
@@ -60,6 +60,21 @@ <h4>{% trans "Calendar" %}</h4>
         {% endcomment %}
     </div>
     <div class="col-sm-10" id="course-content">
+        {% if instance.is_past and instance.has_enrollment_closed %}
+            <div class="alert sticky-alert alert-default">
+                {% trans "This course has already ended. " %}
```

Comment:
The space at the end of the string doesn't seem to be importart. The new line will tell browsers to use a space between text and the next element.

Reply:
Oh nice, good to know. The space has now been removed.